src/utils/source-maps.js

```diff
@@ -16,7 +16,7 @@ export async function getGeneratedLocation(
 
   const { line, sourceId, column } = await sourceMaps.getGeneratedLocation(
     location,
-    source
+    source.toJS()
```

Comment:
Was this intended?

Reply:
Yes, it was - moving the toJS closer to the source.